system_tests/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM google/python
+
+# Inject secrets
+# TODO: Encrypt/decrypt secrets
+ADD data/ data/
+ENV GOOGLE_APPLICATION_CREDENTIALS /home/vmagent/app/data/cloud-python-runtime-qa-credentials.json
```

Comment:
I *think* /home/vmagent/app is symlinked to /app. It might be easier to just use that path.